client/src/app/auth/register/register.component.scss

```diff
@@ -0,0 +1,246 @@
+html {
```

Comment:
If I am not wrong, this file is a copy of login.component.scss. I recommend creating one file with styles for both components and linking it in `@Component({styleUrls: []})`